protocols/gossipsub/src/error.rs

```diff
@@ -36,6 +36,9 @@ pub enum PublishError {
     MessageTooLarge,
     /// The compression algorithm failed.
     TransformFailed(std::io::Error),
+    /// Messages could not be sent because all queues for peers were full. The usize represents the
+    /// number of peers that have full queues.
```

Comment:
```suggestion
    /// Messages could not be sent because the queues for all peers were full. The usize represents the
    /// number of peers that were attempted.
```